.travis.yml

```diff
@@ -1,11 +1,39 @@
-language: minimal
+language: generic
 
-install:
-  - ./build.sh
+matrix:
+  include:
+    - os: linux
+      env:
+        - DOCKERIMAGE=condaforge/linux-anvil-aarch64
+        - DOCKERIMAGE_TEST=arm64v8/centos:7
```

Comment:
This image doesn't have qemu, and therefore won't run on Travis

Reply:
Oh, I see now! Thanks for pointing that out I didn't notice it wasn't running on Travis. So I guess I'll go back to the previous situation where we use `condaforge/linux-anvil-aarch64` for testing.

Reply:
Or I create a custom test image with `ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static`? I don't have strong feelings about what's best.

Reply:
yeah, maybe just create a bare image with qemu. I personally want to add a list of images to test on. Many users use strange images, and it would be good to have a matrix to test against.

Reply:
So testing on Ubuntu 16.04,Ubuntu 18.04, Debian, it will be quick, and allow us to knock out common problems.

Reply:
I'll create a base test image and in the future, we could create more "strange images" to test against if needed.

Reply:
I assume the creatioj and installation will just happen during the build process. It isn't a large addition to the base image.